wiki/Community/Forum/en.md

```diff
@@ -0,0 +1,71 @@
+---
+tags:
+  - forums
+  - subforum
+---
+
+# Forum
+
+*See also: [Internet forum](https://en.wikipedia.org/wiki/Internet_forum)*
+
+[The osu! **forums**](https://osu.ppy.sh/community/forums) are a place for conversations where messages are grouped by subjects, and no quick response is expected. The discussions are searchable, stored forever, and may be revisited at any time, which makes them different from the near-instant messages in [osu! chat](/wiki/Community/Internet_Relay_Chat). As a result, forum messages are longer, contain more information, and typically provide more details than chat messages would.
+
+The osu! forums provide [features](#features) similar to most internet forums: [BBCode](/wiki/BBCode) markup, polls, pins and subscriptions, private messages, and [moderation](#moderation).
+
+## Structure
+
+Each of the osu! forums contains a variety of topics, called **threads**, as well as smaller categories, or **subforums**. Most of the forums are English only, but there also exists a number of [language-specific](https://osu.ppy.sh/community/forums#forum-23) forums for local communities that are large enough.
```

Comment:
from what i can tell "topics" is the sole descriptor in use on the website, so that should be the bold part. maybe `sometimes called "threads"` if you think that's important lingo to mention

Reply:
and pretty much same thing for all of ctrl+f "thread"